Resources/Textures/DeltaV/Clothing/Ears/Headsets/legal.rsi/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/f8f4aeda930fcd0805ca4cc76d9bc9412a5b3428 | Modified by leonardo-dabpeis",
```

Comment:
this needs to point to a github or discord, for all the crediting.

Reply:
The credits point to a discord username. Should they link to a server as well?